src/exe/FreeC/Backend.hs

```diff
@@ -150,10 +150,33 @@ convertModuleToAgda :: IR.Module -> Application String
 convertModuleToAgda =
   fmap showPretty . liftConverter . Agda.Converter.convertModule
 
+-- | Creates an @.agda-lib@ file for the output directory.
+--
+--   The file declares dependencies on the Agda standard library and our base
+--   library.
+createAgdaLib :: Application ()
+createAgdaLib = do
+  (agdaLib, name) <- getAgdaLib
+  liftIO $ do
+    createDirectoryIfMissing True (takeDirectory agdaLib)
+    writeFile agdaLib $ contents name
+ where
+  -- | Creates the string to write to the @.agda-lib@ file.
+  contents :: String -> String
+  contents name =
+    unlines ["name: " ++ name, "include: .", "depend: standard-library base"]
+
+  -- | Path to the @.agda-lib@ file to create and the name of the library.
+  getAgdaLib :: Application (FilePath, String)
+  getAgdaLib = do
+    Just outputDir <- inOpts optOutputDir
```

Comment:
In order to avoid a runtime error, the generation of the `.agda-lib` file must be disabled if there is no `--output` directory. We should also consider to add a `--no-agda-lib` command line option (or generalize the `--no-coq-project` command line option). Also don't overwrite the file if it exists already.